### PR TITLE
Replacing some CultureInfo.CurrentCulture occurrences

### PR DESCRIPTION
**Pull request by anderson-joyle**
Issue #1474.

### CODE REVIEW

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/libraries/Microsoft.PowerFx.Core/Errors/ErrorUtils.cs [hunk 1]
```diff
@@ -30,8 +30,7 @@ internal static string FormatMessage(string message, CultureInfo locale, params
             {
                 try
                 {
-                    // $$$ can't use current culture
-                    sb.AppendFormat(locale ?? CultureInfo.CurrentCulture, message, args);
```
Comment (MikeStall) [inline src/libraries/Microsoft.PowerFx.Core/Errors/ErrorUtils.cs:34]: >CurrentCulture [](http://example.com/codeflow?start=58&length=14)

Can we be confident this change isn't breaking power Apps?

**Review by MikeStall — CHANGES_REQUESTED** [review]
:clock1:

**Comment by MikeStall** [comment]
Can we be confident this change isn't breaking power Apps? PA runs in a dedicated process - does PA rely on setting CurrentCulture anywhere?

**Comment by LucGenetier** [comment]
✅ No public API change.

**Pull request closed without merging**